src/backend/controllers/AddressController.js

```diff
@@ -0,0 +1,148 @@
+import { Response } from "miragejs";
+import { formatDate, requiresAuth } from "../utils/authUtils";
+
+/**
+ * All the routes related to Cart are present here.
+ * These are private routes.
+ * Client needs to add "authorization" header with JWT token in it to access it.
+ * */
+
+/**
+ * This handler handles getting items to user's address.
+ * send GET Request at /api/user/address
+ * */
```

Comment:
remove commented code

Reply:
It's docs for a better understanding of the code.